promql/promqltest/testdata/functions.test

```diff
@@ -965,8 +964,8 @@ eval instant at 1m min_over_time(data[1m])
 
 eval instant at 1m max_over_time(data[1m])
```

Comment:
Same pattern here (essentially, if there is less than a minute of time series data, and the test query is `at 1m ... [1m]`, it probably wants to touch all the samples, but with the new range definition, we now exclude the 1st sample, so we need a longer range. Same for line 958 above. It's only by accident that the results don't change. In both cases, use a longer range, e.g. 2m.